visbrain/io/read_sleep.py

```diff
@@ -131,11 +138,24 @@ def __init__(self, data, channels, sf, hypno, href, preload, use_mne,
                 raise ValueError("Then length of the hypnogram must be the "
                                  "same as raw data")
         if isinstance(hypno, str):  # (*.hyp / *.txt / *.csv)
+            hypno_file = hypno
             hypno, _ = read_hypno(hypno, time=time, datafile=file)
             # Oversample then downsample :
             hypno = oversample_hypno(hypno, self._N)[::dsf]
             PROFILER("Hypnogram file loaded", level=1)
 
+        # set new window title with file name of data and hypnogram
+        _translate = QtCore.QCoreApplication.translate
+        window_title = _translate("MainWindow", "Sleep")
+        if data_file:
+            window_title += ' | {}: {}'.format(_translate("MainWindow",
+                                                          "File"), data_file)
+        if hypno_file:
+            window_title += ' | {}: {}'.format(_translate("MainWindow",
```

Comment:
@skjerns  If provided the full path to the hypnogram / data file, it's not going to be a really long title? Might use something like `os.path.split` to only have the file name?

Reply:
Sometimes the edf names are generic (eg "recording.edf") and only the folder gives the information which file is loaded. Therefore, I included the whole directory.

One compromise would be to display the curr. folder name and the file, e.g `folder/recording.edf` instead of `c:/data/folder/recording.edf`.

Else I can also only display the filename, like you suggested.
Let me know what you think!

Reply:
Ok, good point. 

> One` compromise would be to display the curr. folder name and the file, e.g patient_x/data.edf instead of c:/data/patient_x/data.edf.

Yes that could be an idea. For the hypnogram, do you think we should conserve the full path? Or only the file name could be sufficient?

@raphaelvallat your opinion?

Reply:
I'd conserve the folder name for both files. Too often have I misclicked on the wrong folder and loaded a wrong hypnogram and only realized later...

Reply:
Agree with @skjerns, I would show parent directory + filename!

Reply:
Ok for me